protobufs/funding/funding.proto

```diff
@@ -42,31 +42,31 @@ service FundingService {
   }
   rpc Get(Request) returns (FundingResponse) {
     option (google.api.http) = {
-      get: "/documents/{identifier}/funding/{funding_id}"
+      get: "/document/{identifier}/funding/{funding_id}"
```

Comment:
This should still be `documents` followed by document identifier

Reply:
For the proofs endpoint, we calling it `document` that's the reason why I changed it.

I don't mind we only need to be consistent

Reply:
Some endpoints will change, especially those that dont follow REST standard. I would rather us start with this instead of changing it again. Lets go with documents here and proofs we will change later